core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.html

```diff
@@ -1,19 +1,38 @@
 <ng-container *ngIf="vm$ | async as vm">
     <dot-edit-ema-toolbar>
-        <h4 title>{{ vm.pageTitle }}</h4>
         <ng-container left>
-            <div class="dummy-button"></div>
-            <div class="dummy-button"></div>
-            <div class="dummy-button"></div>
-            <div class="dummy-button"></div>
+            <p-button label="Preview" icon="pi pi-desktop" styleClass="p-button-text p-button-sm" />
+            <p-button label="Bookmark" icon="pi pi-star" styleClass="p-button-text p-button-sm" />
+            <p-button
+                [cdkCopyToClipboard]="vm.iframeUrl"
```

Comment:
This directive have a `copied` event, check here: https://material.angular.io/cdk/clipboard/api#directives